cookbooks/web/recipes/cgimap.rb

```diff
@@ -124,7 +57,7 @@
   service "cgimap" do
     action [:enable, :start]
     supports :restart => true, :reload => true
-    subscribes :restart, "execute[cgimap-build]"
+    subscribes :restart, "dpkg_package[openstreetmap-cgimap-bin]"
```

Comment:
Is `dpkg_package` required here? or will `package` work? Not sure we have any previous example...